src/chart/cartesianChart.js

```diff
@@ -136,14 +154,21 @@ export default function(xScale, yScale) {
 
             // set the axis ranges
             var plotAreaContainer = svg.select('.plot-area');
-            xScale.range([0, plotAreaContainer.layout('width')]);
-            yScale.range([plotAreaContainer.layout('height'), 0]);
+            setScaleRange(xScale, [0, plotAreaContainer.layout('width')]);
+            setScaleRange(yScale, [plotAreaContainer.layout('height'), 0]);
 
             // render the axes
-            var axesContainer = svg.select('.axes-container');
-            axesSeries.xScale(xScale)
-                .yScale(yScale);
-            axesContainer.call(axesSeries);
+            xAxis.xScale(xScale)
+                .yScale(isOrdinal(yScale) ? d3.scale.identity() : yScale);
```

Comment:
Would it be possible to always make the axes accept an identity axis in the cross direction?